services/bundle_analysis.py

```diff
@@ -25,3 +31,108 @@ def load_report(
         repo_key=ArchiveService.get_archive_hash(commit.repository),
     )
     return loader.load(commit_report.external_id)
+
+
+def load_time_conversion(size):
+    """
+    Converts total size in bytes to approximate time (in seconds) to download using a 3G internet (3 Mbps)
```

Comment:
Did we decide on 3G as a standard?

Reply:
I don't think we did a full discussion around what download speed we should do, but probably the most widely used speed. I envision in future iterations we can provide as input what the preferred speed is to make the calculation. But this should suffice for now in the v0.